go/vt/vtctl/vtctl.go

```diff
@@ -314,7 +314,7 @@ var commands = []commandGroup{
 				"<from_keyspace> <to_keyspace> <tables>",
 				"Start the VerticalSplitClone process to perform vertical resharding. Example: SplitClone from_ks to_ks 'a,/b.*/'"},
 			{"VDiff", commandVDiff,
-				"-workflow=<workflow> <target keyspace> [-source_cell=<cell>] [-target_cell=<cell>] [-tablet_types=REPLICA] [-filtered_replication_wait_time=30s]",
+				"[-source_cell=<cell>] [-target_cell=<cell>] [-tablet_types=replica] [-filtered_replication_wait_time=30s] <keyspace.workflow>",
```

Comment:
Do we allow `.` in keyspace names or workflow names? If we don't, do we validate the names when they are created?

Reply:
In mysql, a `.` in the schema name is a bad idea because it's a separator for `db.table`, but I'm assuming you can escape it with a backtick. We should eventually tighten all this. But there are many places where we don't check (or escape) well.

For now, we rely on the good behavior of our users :).